test/unit/util/test_translator.py

```diff
@@ -53,3 +57,85 @@ def translator_response(
 def test_translator_converts_response_to_correct_type(response_type):
     response, object_class = _response_to_class_mapping[response_type]
     assert type(Translator().translate(response.json()['type']) == object_class)
+
+
+def test_default_translator():
+    assert isinstance(Translator._default_translator, Translator)   # pylint:disable=protected-access
+
+
+@pytest.mark.parametrize(('extend_default_translator', 'new_child'), list(product([None, True], [False, True])))
```

Comment:
Normally we would make fixture functions here, one for `extend_default_translator`, and another for `new_child`.